server/routes/basicDetails.ts

```diff
@@ -240,12 +240,15 @@ export default function basicDetailsRoutes(
 
   function findDefaultAddressInAddressList(addressList: Array<DeliusAddress>): DeliusAddress {
     let defaultAddress: DeliusAddress = null
```

Comment:
Would something like this be simpler?
```suggestion
  function findDefaultAddressInAddressList(addressList: Array<DeliusAddress>): DeliusAddress {
    return (
      addressList.find(a => a.status === 'Default') ??
      addressList.find(a => a.status === 'Postal') ??
      addressList.find(a => a.status === 'Main')
    )
  }
```

or even this:
```suggestion
  function findDefaultAddressInAddressList(addressList: Array<DeliusAddress>): DeliusAddress {
    return addressList.find(a => ['Default', 'Postal', 'Main'].includes(a.status))
  }
```

Reply:
Updated with suggestion 1 thanks